zenith_utils/src/pq_proto.rs

```diff
@@ -346,7 +346,7 @@ pub enum BeMessage<'a> {
     DataRow(&'a [Option<&'a [u8]>]),
     ErrorResponse(String),
     // see https://www.postgresql.org/docs/devel/protocol-flow.html#id-1.10.5.7.11
-    Negotiate,
+    Negotiate(bool),
```

Comment:
Looks like the PostgreSQL docs don't give a name to the `Y`/`N` response to an SSLRequest. `Negotiate` doesn't make it very clear what it is. I'd suggest `SSLHandshakeResponse` or something, with a comment pointing out it doesn't have a name in PostgreSQL.

I'm not sure how stable the link to the doc is, I think the anchors can change between PostgreSQL (major) versions. So in a year or two, that might point to wrong section.